src/quests/prologue/tasks/train.ts

```diff
@@ -0,0 +1,69 @@
+import { Task } from "grimoire-kolmafia";
+import { use } from "kolmafia";
+import { $item, TrainSet, get, have } from "libram";
+
+import { TRAIN } from "../../../prefs/properties";
+import { CMCInstalled } from "./cmc";
+
+function trainInstalled(): boolean {
+  if (!get(TRAIN)) {
+    return true;
+  }
+  if (TrainSet.installed()) {
+    return true;
+  }
+  return !have($item`model train set`);
+}
+
+const defaultPieces = [
+  TrainSet.Station.COAL_HOPPER,
+  TrainSet.Station.CANDY_FACTORY,
+  TrainSet.Station.ORE_HOPPER,
+  TrainSet.Station.TRACKSIDE_DINER,
+  TrainSet.Station.GRAIN_SILO,
+  TrainSet.Station.TOWER_FIZZY,
+  TrainSet.Station.GAIN_MEAT,
+  TrainSet.Station.VIEWING_PLATFORM,
+];
+
+export const installTrain: Task = {
+  name: `Install Train`,
+  completed: () =>
+    get(`_workshedItemUsed`) ||
+    trainInstalled() ||
+    (CMCInstalled() && get(`_coldMedicineConsults`) < 5),
```

Comment:
Let's keep CMC out of this method. With 3 worksheds supported, I think it makes sense to refactor the `useAsdon` property to a `worshed` property, accepting `train,cmc,asdon`.